src/Sparrow/Logging/LoggingSource.cs

```diff
@@ -362,6 +425,8 @@ private class LocalThreadWriterState
         {
             public readonly ForwardingStream ForwardingStream;
 
+            public readonly List<WebSocket> BlockedWebSockets = new List<WebSocket>();
```

Comment:
What is this for? You are going to be adding to this all the time, why not just do that on the entry when you call `WriteToListeningWebSockets`